chemie/corporate/views.py

```diff
@@ -0,0 +1,114 @@
+from django.shortcuts import render, get_object_or_404
+from django.shortcuts import redirect, reverse
+from django.utils import timezone
+from django.contrib.auth.decorators import permission_required
+
+from .models import Interview, JobAdvertisement
+
+from chemie.committees.models import Committee
+from chemie.events.models import Bedpres, Social
+from .forms import CreateInterviewForm, CreateJobForm
+
+
+def index(request):
+    indkom = Committee.objects.get(title="Industrikomiteen")
```

Comment:
This would cause an error if the name of the committee is ever changed.

Reply:
Let's unngå that